README.md

```diff
@@ -233,4 +195,41 @@ Other WDL examples on Cromwell on Azure:
 
 ## Related Projects
 
-[Genomics Data Analysis with Jupyter Notebooks on Azure](https://github.com/microsoft/genomicsnotebook)<br/>
+[Genomics Data Analysis with Jupyter Notebooks on Azure](https://github.com/microsoft/genomicsnotebook)<br/>
+
+### How to build the deployment executable yourself

```

Comment:
Make it header2 instead of header3? Now it looks like a part of Related Projects section